handlers/namespaces.go

```diff
@@ -115,10 +116,12 @@ func NamespaceUpdate(w http.ResponseWriter, r *http.Request) {
 	}
 	jsonPatch := string(body)
 
-	// TODO: Multicluster: Add query parameter
-	cluster := params["cluster"]
-	if cluster == "" {
-		cluster = kubernetes.HomeClusterName
+	query := r.URL.Query()
```

Comment:
There is a helper method to use in the handlers package called: `clusterNameFromQuery`, it does this logic already!.

Reply:
Thanks, @leandroberetta ! Very useful!